### DISCUSSION TITLE
How to access process.env.MODE inside a Vue template?

### DISCUSSION THREAD

**Discussion by gustavotoyota** [quasarframework/quasar#15099] [General - Components / Directives / etc] [answered]
I tried to use process.env.MODE on the client.
It works on development, but on production it throws `ReferenceError: process is not defined`.

How to identify the Quasar mode on the client?

**Comment by TobyMosque** [comment, marked as answer]
expose the mode in the setup:
```html
<template>
  <div>{{mode}}</div>
</template>
```
```ts
<script setup lang="ts">
defineExpose({
  mode: process.env.MODE
})
<script>
```

**Reply by gustavotoyota** [reply]
I'm an idiot 😅, I forgot that the process.env works by "replacing" values.
I modified your example to use a simple const:
```js
<script setup lang="ts">
const quasarMode = process.env.MODE
<script>
```
